Makefile

```diff
@@ -68,6 +68,15 @@ lint/gomod: tidy
 		exit 1; \
 	fi
 
+lint/buf: fmt/buf
```

Comment:
Simply running `buf lint` raises too many errors and is not usable. So I'd like to run buf fmt and check if any diff exists instead at least for now.